site/en/r1/guide/checkpoints.md

```diff
@@ -168,9 +165,6 @@ In other words, as the following illustration suggests, once checkpoints
 exist, TensorFlow rebuilds the model each time you call `train()`,
 `evaluate()`, or `predict()`.
 
-<div style="width:80%; margin:auto; margin-bottom:10px; margin-top:20px;">
-<img style="width:100%" src="../images/subsequent_calls.png">
```

Comment:
And this to https://www.tensorflow.org/images/subsequent_calls.png

Reply:
Reverse the image tag and apply it.
Thanks